projects/make-a-vis/src/lib/data-view/start-project-options/start-project-options.component.ts

```diff
@@ -9,19 +9,27 @@ import { LoadProjectService, ProjectExtensionType } from '../../shared/services/
   styleUrls: ['./start-project-options.component.scss']
 })
 export class StartProjectOptionsComponent implements OnInit {
-
+  @ViewChildren('startProjectFileInputTag') fileInputTags: QueryList<ElementRef>;
   projectExtensions: ProjectExtensionType[] = ['yml', 'nsf', 'isi', 'csv'];
 
   constructor( public loadProjectService: LoadProjectService) { }
 
   ngOnInit() {
   }
 
-  readNewFile(event: any, selectedExtension: ProjectExtensionType) {
+  readNewFile(event: any, target: any, selectedExtension: ProjectExtensionType) {
```

Comment:
You have target in event. Why pass it in twice?

Reply:
It's because when the mat menu collapses, we lose the target element, so i had to pass it as an argument.